firefly_widgets/table.py

```diff
@@ -18,6 +18,9 @@ class TableViewer(widgets.DOMWidget):
     url_or_path = Unicode().tag(sync=True)
     data_url = Unicode().tag(sync=True)
     tbl_group = Unicode().tag(sync=True)
+    conn_id = Unicode().tag(sync=True)
+    channel = Unicode().tag(sync=True)
+    tbl_id = Unicode('tableViewer-1').tag(sync=True)
```

Comment:
tbl_id is unique identifier of the table. Do we want all widgets created without tbl_id attribute refer to the same table?